apps/examples/mtdpart/Kconfig

```diff
@@ -0,0 +1,50 @@
+#
+# For a description of the syntax of this configuration file,
+# see the file kconfig-language.txt in the NuttX tools repository.
```

Comment:
You had better use below.
# see kconfig-language at https://www.kernel.org/doc/Documentation/kbuild/kconfig-language.txt